packages/extensions-sdk/src/cli/commands/validators/check-built-code.ts

```diff
@@ -0,0 +1,65 @@
+import { EXTENSION_PKG_KEY } from '@directus/extensions';
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import path from 'path';
+import type { Report } from '../../types.js';
+import { isObject } from '@directus/utils';
+
+const checkBuiltCode = {
+	name: 'built-code',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for built code';
+
+		let codePath: any = '/dist';
+		const packagePath = path.resolve('package.json');
+
+		if (await fse.pathExists(packagePath)) {
+			const packageFile = await fse.readJson(packagePath);
+
+			if (packageFile[EXTENSION_PKG_KEY]) {
+				const { path } = packageFile[EXTENSION_PKG_KEY];
+
+				if (path) {
+					const message = `Path ${path} found in ${EXTENSION_PKG_KEY}`;
+					spinner.text = message;
+
+					reports.push({
+						level: 'info',
+						message: `${checkBuiltCode.name}: ${message}`,
+					});
+
+					codePath = path;
+				}
+			}
+		}
+
+		if (!isObject(codePath)) {
```

Comment:
I think we could remove the dependency on `isObject`.

```suggestion
		if (typeof codePath === 'string') {
```